centreon/src/Centreon/Domain/Repository/ContactGroupRepository.php

```diff
@@ -49,46 +62,59 @@ public function getPaginationList($filters = null, int $limit = null, int $offse
     {
         $collector = new StatementCollector();
 
-        $sql = 'SELECT SQL_CALC_FOUND_ROWS * FROM `' . ContactGroup::TABLE . '`';
+        $sql = 'SELECT SQL_CALC_FOUND_ROWS * FROM `:db`.contactgroup';
 
         $isWhere = false;
         if ($filters !== null) {
-            if (array_key_exists('search', $filters) && $filters['search']) {
+            if (
+                array_key_exists('search', $filters)
+                && $filters['search']
+            ) {
                 $sql .= ' WHERE `cg_name` LIKE :search';
                 $collector->addValue(':search', "%{$filters['search']}%");
                 $isWhere = true;
             }
-            if (array_key_exists('ids', $filters) && is_array($filters['ids'])) {
+            if (
+                array_key_exists('ids', $filters)
+                && is_array($filters['ids'])
+                && [] !== $filters['ids']
+            ) {
                 $idsListKey = [];
                 foreach ($filters['ids'] as $x => $id) {
                     $key = ":id{$x}";
                     $idsListKey[] = $key;
-                    $collector->addValue($key, $id, PDO::PARAM_INT);
+                    $collector->addValue($key, $id, \PDO::PARAM_INT);
                     unset($x, $id);
                 }
                 $sql .= $isWhere ? ' AND' : ' WHERE';
                 $sql .= ' `cg_id` IN (' . implode(',', $idsListKey) . ')';
             }
         }
 
-        if (!empty($ordering['field'])) {
+        if (! empty($ordering['field'])) {
```

Comment:
For homogeneity with previous
```suggestion
        if ($ordering['field'] ?? false) {
```

Reply:
done ✅